Websites/webkit.org/demos/webgpu/scripts/hello-cube-auto-layout.js

```diff
@@ -76,20 +76,6 @@ async function helloCube() {
     });
 
     /*** Shader Setup ***/
-    
```

Comment:
I think modifying these files will actually push updates to the real live site. Is that really what we want?

Reply:
It should be fine, since the feature is still compiled out so we will see the same behavior on Safari and STP as before (WebGPU is not supported). Unless we don’t want the files accessible, but I think you can only reach many of the html files if you were to know the full path as the index.html file is unchanged.